chord_metadata_service/chord/data_types.py

```diff
@@ -9,6 +9,7 @@
 
 DATA_TYPE_EXPERIMENT = "experiment"
 DATA_TYPE_PHENOPACKET = "phenopacket"
+DATA_TYPE_FHIR = "fhir"
```

Comment:
We're still importing "phenopacket" data with FHIR since it gets translated - unless we want to consider tables stored in the FHIR format to be somehow distinct, in which case we can make a fhir app with it's own models

Reply:
Ok I didn't get this part, so I can remove DATA_TYPE_FHIR at all.